File: Iniciante/URI 1013.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def valor1 (Integer/parseInt (get line 0)))
(def valor2 (Integer/parseInt (get line 1)))
(def valor3 (Integer/parseInt (get line 2)))

(cond
    (and (> valor1 valor2) (> valor1 valor3)) 
        (printf "%d eh o maior\n" valor1)

    (> valor2 valor3) 
        (printf "%d eh o maior\n" valor2)

    :else 
        (printf "%d eh o maior\n" valor3)
)

File: Iniciante/URI 1013.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
valor1 = as.integer(line[[1]][1])
valor2 = as.integer(line[[1]][2])
valor3 = as.integer(line[[1]][3])

if (valor1 > valor2 && valor1 > valor3) {
    write(sprintf("%d eh o maior", valor1), '')
} else if (valor2 > valor3) {
    write(sprintf("%d eh o maior", valor2), '')
} else {
    write(sprintf("%d eh o maior", valor3), '')
}